test/Interop/Cxx/enum/anonymous-with-swift-name.swift

```diff
@@ -0,0 +1,34 @@
+// RUN: %target-run-simple-swift(-I %S/Inputs -Xfrontend -enable-experimental-cxx-interop)
+
+// REQUIRES: executable_test
+
+import AnonymousWithSwiftName
+import StdlibUnittest
+
+var AnonymousEnumsTestSuite = TestSuite("Anonymous Enums With Swift Name")
+
+AnonymousEnumsTestSuite.test("SOME_OPTIONS") {
+  let red: SOColorMask = .red
+  let green: SOColorMask = .green
+  let blue: SOColorMask = .blue
+  let all: SOColorMask = .all
+
+  expectEqual(red.rawValue, 2)
+  expectEqual(green.rawValue, 4)
+  expectEqual(blue.rawValue, 8)
+  expectEqual(all, .all)
```

Comment:
This test seems tautological. (And if there's any case I'd be worried about getting the wrong raw value for, it's this one.) What gives?

Reply:
Yeah, I guess I can do `~UCUnsignedInt(0)`. Was worried it wouldn't work/be the same value on Swift, but thinking more about it, it should be.